core-bundle/src/File/Metadata.php

```diff
@@ -68,6 +87,11 @@ public function getUrl(): string
         return $this->values[self::VALUE_URL] ?? '';
     }
 
+    public function getUuid(): ?string
```

Comment:
Should we add a doccomment that tells us if it is a binary 16-byte UUID or in the ASCII format?

Reply:
Is the one inside `FilesModel->uuid` always in ASCII format or do we need to enforce this when setting it?

Reply:
I think `FilesModel->uuid` is always in binary format.

Reply:
Let's enforce ASCII

Reply:
If we plan to use objects for UUIDs in the future we could return an uuid object here?
`\Symfony\Component\Uid\Uuid::fromString(...)`

Reply:
Done in 121f44875077ab1dce100ca2b1ed9245f5e9ff74 + e4a7bfee65eb89354e5614bc4807b2a6dc2c894b.